src/index.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
Is renaming this necessary?

Reply:
Not necessary but common and I like it short :)